templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
I don't think we want to code `<nav>` elements in to the footer.  The assumption is that everything in this region is navigation, and that's not necessarily the case, or at least, is not suggested by the region name, I think we should consider:

1: Moving the `<nav>` element into the menu block template (I assume that't what we're anticipating going in here) - then we catch all menu blocks (including sidebar), which is a bonus.

or

2: Renaming this region `footer-navigation` to make it more explicit what type of content this region is designed to have in it.

Thoughts?